NewForce-Cohort-6/tabloid-cli-coven-of-the-midnight-sun
# Ticket 16 list posts

# Description

https://trello.com/c/XmkqSEi0/16-list-posts-16

This adds List Post functionality to PostRepository and PostManager.

If PR for Ticket 15 introduced build error bug, this fixes bug.


# Testing Instructions
Use branch ntListPost

Navigate steps:
Main Menu > Post Management > List Posts

You should see at least one post listed: "Forms of notational..."
If you tested ticket 15, you may also see posts that you added while testing that PR.

# Checklist:
